AnalyzingAndProfilingTools/ProfileSample/ProfileSample/Controllers/HomeController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Mvc;
+using ProfileSample.DAL;
+using ProfileSample.Models;
+
+namespace ProfileSample.Controllers
+{
+    public class HomeController : Controller
+    {
+        public async Task<ActionResult> Index()
+        {
+
+            //var sources = context.ImgSources.Take(20).Select(x => x.Id);
+
+            //foreach (var id in sources)
+            //{
+            //    var item = context.ImgSources.Find(id);
+
+            //    var obj = new ImageModel()
+            //    {
+            //        Name = item.Name,
+            //        Data = item.Data
+            //    };
+
+            //    model.Add(obj);
+            //} 
+
+            var temp = await Convert(); // First for added images db
+
+            var model = new List<ImageModel>();
+            var context = new ProfileSampleEntities();

```

Comment:
ef db context is disposable you have to use using statement https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/using